easybuild/toolchains/gcccore.py

```diff
@@ -35,6 +35,9 @@
 from easybuild.tools import LooseVersion
 from easybuild.tools.toolchain.toolchain import SYSTEM_TOOLCHAIN_NAME
 
+# At GCCcore level we do not allow OpenMP
+GCCCORE_BANNED_LIBRARIES = ['libgomp.so']
```

Comment:
Has there been a discussion of what version of `GCCcore` we should add this check for? I do not think it is worthwhile to apply this to all `GCCcore` versions we have.

Probably start with one of:
* `14.2.0` (`2025a` generation)
* `14.3.0` (`2025b` generation)

Reply:
That's true, I think it only really matters once we introduce modern LLVM (although it was probably also a potential issue for Intel compilers since we introduced `GCCcore`)